### PR TITLE
do_build: add haskell & ocaml layers.

### PR DESCRIPTION
**Pull request by eric-ch**
Just in case someone has a legacy builder using the deprecated do_build script.

### CODE REVIEW

**Comment by rossphilipson** [comment]
LGTM. Since you are the only one using it I will merge it soon :)

File: setup_build [hunk 1]
```diff
@@ -113,6 +118,8 @@ if [ "$1" != "env" ]; then
   getgit $REPOS/meta-openembedded $META_OE_REPO $META_OE_TAG
   getgit $REPOS/meta-java $META_JAVA_REPO $META_JAVA_TAG
   getgit $REPOS/meta-selinux $META_SELINUX_REPO $META_SELINUX_TAG
+  getgit $REPOS/meta-openxt-ocaml-platform $META_OPENXT_OCAML_PLATFORM_REPO $META_OPEMXT_OCAML_PLATFORM_TAG
```
Comment (dozylynx) [inline setup_build]: There's a typo in this line: META_OPEMXT_OCAML_PLATFORM_TAG needs correcting!

File: setup_build [hunk 2]
```diff
@@ -63,6 +63,11 @@ META_JAVA_TAG=5d528acac0eab6cfaa4ed3c290c7db3acd5047bf
 META_SELINUX_REPO=git://git.yoctoproject.org/meta-selinux
 META_SELINUX_TAG=8e952c7da126d30f4aecb33abd3ef9252785cb40
 
+META_OPENXT_OCAML_PLATFORM_REPO=https://github.com/OpenXT/meta-openxt-ocaml-platform.git
+META_OPEMXT_OCAML_PLATFORM_TAG=master
```
Comment (dozylynx) [inline setup_build]: same typo here

**Comment by dozylynx** [comment]
thanks for doing this

**Comment by eric-ch** [comment]
@dozylynx oops! fixing... No problem, if you had something else almost ready for do_build I can close this one btw.

**Comment by rossphilipson** [comment]
Fixed so merging....

**Pull request merged**